include/envoy/network/io_handle.h

```diff
@@ -268,8 +282,31 @@ class IoHandle {
    *               should initially listen on.
    * @return @ref Event::FileEventPtr
    */
-  virtual Event::FileEventPtr createFileEvent(Event::Dispatcher& dispatcher, Event::FileReadyCb cb,
-                                              Event::FileTriggerType trigger, uint32_t events) PURE;
+  virtual Event::FileEventPtr createManagedFileEvent(Event::Dispatcher& dispatcher,
```

Comment:
Consider renaming this method to avoid in being used in cases where initializeFileEvent is preferable.  My name suggestion is: createFileEventForListenSocket

Reply:
No longer applicable since we moved to duplicating the socket